### DISCUSSION TITLE
Just wanted to appreciate

### DISCUSSION THREAD

**Discussion by sultank1** [nasirkhan/laravel-starter#334] [General]
hi nasir  thank you very much for building such versatile starter for laravel.... 
it has save me a lot of time.
i have used it in couple of project already .... 

keep it up ... 

good bless you ...

**Comment by nasirkhan** [comment]
Great to hear from you! 
I am very happy that you used LaravelStarter for your projects. 

Please do mention if you think this could be improved by adding new features.

**Reply by sultank1** [reply]
thanks for asking . sure i will do so .. already it more than enough for me ...  thanks
